libs/MobileSync/src/com/salesforce/androidsdk/mobilesync/manager/LayoutSyncManager.java

```diff
@@ -59,7 +59,7 @@
 
     static final String SOUP_NAME = "sfdcLayouts";
     static final String QUERY = "SELECT {" + SOUP_NAME + ":_soup} FROM {" + SOUP_NAME +
```

Comment:
What if the sync down target was created before 8.2? That query won't find it.

Reply:
Yes, to avoid the upgrade step, I figured we'd pay the one-time cost of an extra sync. It'll fallback on server and hydrate the cache if someone switches from the old API to the new one.